execution/distinct.go

```diff
@@ -116,6 +120,7 @@ func (rs *recordSet) Insert(r *Record) (bool, error) {
 	return false, nil
 }
 
+//TODO: should the hash include other fields as well?
 func HashRecord(rec *Record) (uint64, error) {
-	return hashstructure.Hash(rec.data, nil)
+	return hashstructure.Hash(octosql.GetValuesFromPointers(rec.Data), nil)
```

Comment:
Distinct shouldn't grab into internal fields.

Reply:
changed it so that Hash is now a method implemented on record (record.Hash)